### PR TITLE
adding welcome doc draft

### PR DESCRIPTION
**Pull request by charity**

### CODE REVIEW

File: 0000-charity-welcome-doc.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Introducing a new series: Post-Mortem Book Reports
+
+## Dear fellow systems engineers,
```
Comment (pshima) [inline 0000-charity-welcome-doc.md:3]: Should this be addressed to "systems engineers" or something more generic like operators?

File: 0000-charity-welcome-doc.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+A couple of book reports are already up: see [@petey5k's piece on the recent GitHub outage](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews/blob/master/2016-01-28-pshima-github-outage.md), and [@gabinante's review of the classic 2011 EBS outage](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews/blob/master/2016-03-07-gabinante-AWS-EBS-2011.md).  
+
+We appreciate the effort that everyone puts into post mortems and public retrospectives.  Writing retrospectives is hard!  Pitching the level of detail to the right audience is hard.  We hope that providing feedback on how your retrospectives were meaningful for us will give everyone more insight into how their efforts are valued and received.
+
+Yours in triumph and in tragedy,
+
+[charity.](https://github.com/charity)
+
+P.S. Want to write a book report?  Please do!!  Check out the [template](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews) and send us a pull request.  :)
```
Comment (pshima) [inline 0000-charity-welcome-doc.md:32]: Should this link to a CONTRIBUTING (that doesn't exist yet)?  Just wondering if the template is clear enough for everyone to get started.

Comment (charity) [inline 0000-charity-welcome-doc.md:32]: @pshima what are you thinking should be in the CONTRIBUTING doc?  should we move the template there?

File: 0000-charity-welcome-doc.md (hunk 1 above)
Comment (charity) [inline 0000-charity-welcome-doc.md:3]: Hmmm, I felt like "systems engineers" was more generic than operators.  Trying to encompass an audience of software engineers, DBAs, SREs, execs, basically anyone interested in systems quality and incident response.  Anyone else have thoughts?

Comment (pshima) [inline 0000-charity-welcome-doc.md:3]: Makes sense, maybe just engineers - either way systems engineers works for me just don't want to exclude anyone.

File: 0000-charity-welcome-doc.md (hunk 2 above)
Comment (pshima) [inline 0000-charity-welcome-doc.md:32]: I think that may be better for people just entering the repo, I can do a PR for the contributing doc in the next couple days.  Also fine to ship as is, just a very minor comment.

Comment (charity) [inline 0000-charity-welcome-doc.md:32]: Awesome.  Think this would be a great change.

**Pull request merged**